### PR TITLE
Add links to videos to README

### PR DESCRIPTION
**Pull request by pmorie**
Add a couple links to videos to the README

### CODE REVIEW

**Comment by arschles** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ services like message brokers that exists _somewhere_ in a simple way:
     application configuration primitives in Kubernetes: Services, Secrets, and
     ConfigMaps.
 
+Video links:
```
Comment (duglin) [inline README.md]: Can we either move the youtube link above down here, or move these up - it seems like they should be next to each other.

Comment (pmorie) [inline README.md]: You're asking to feature the links more prominently?

Comment (duglin) [inline README.md]: I just don't think they should be spread out with some video links up top and others down below.  Just want them all in one spot - no opinion on where in the doc.

**Comment by pmorie** [comment]
Ahh, I totally forgot that we had a link to the youtube playlist.

**Comment by duglin** [comment]
LGTM

@pmorie rebase needed

**Comment by arschles** [comment]
merging now

**Pull request merged**